content/sensu-go/5.2/getting-started/faq.md

```diff
@@ -29,7 +29,8 @@ See the [hardware requirements guide][5] for minimum and recommended hardware to
 
 > Is there an enterprise version of Sensu Go?
 
-Yes! Sensu Go for enterprise is coming soon. Check out this [Sensu blog post](https://blog.sensu.io/announcing-the-sensu-roadmap-sensu-go-release-date) for more information about Sensu Go for enterprise.
+[Yes!](https://blog.sensu.io/enterprise-features-in-sensu-go) Sensu Inc. offers support packages for Sensu Go as well as license-activated features designed for enterprises, like LDAP authentication and support for industry-standard tools.

```

Comment:
extremely minor nit -- the feature elaboration after "enterprises" sounds like we're describing the enterprises. is there a way to shuffle?